src/bin/cql-stress-cassandra-stress/settings/option/column.rs

```diff
@@ -57,9 +61,41 @@ impl ColumnOption {
     }
 }
 
+/// A type for parsing `-col n=` parameter.
+///
+/// In cassandra-stress, CLI originally accepts a distribution.
+/// However, only FIXED(?) distribution is supported by the tool.
+///
+/// It would make much more sense to accept simply a u64 value instead of distribution.
+/// OTOH, we want cql-stress CLI to be compatible with original c-s CLI.
+/// This is why this type is introduced, so the users can choose between
+/// providing either u64 value or FIXED(?) distribution.
+struct ColumnCount;
+
+impl Parsable for ColumnCount {
+    type Parsed = u64;
+
+    fn parse(s: &str) -> Result<Self::Parsed> {
+        let parse_u64_result = u64::parse(s);
+        if parse_u64_result.is_ok() {
+            return parse_u64_result;
+        }
+
+        || -> Result<u64, anyhow::Error> {
+            let s = &s.to_lowercase();
+            let description = parse_description(s, SyntaxFlavor::Classic)
+                .context("Failed to parse distribution description.")?;
+            anyhow::ensure!(description.name == "fixed", "Expected FIXED distribution.");
+            description.check_argument_count(1)?;
+            u64::parse(description.args[0]).context("Failed to parse u64 value")
+        }()
```

Comment:
What a nice trick for seamless use of `?` operator - immediately called closure. Have you been using this elsewhere as well yet?

Reply:
I think I've used the trick once or twice before in cql-stress in very similar context - I wanted to attach some common error message at the end of the function in case of failure. This way we don't have to introduce an additional function/method or variable (closure which has to be then called).